pkg/api/common/versions.go

```diff
@@ -342,29 +331,29 @@ func RationalizeReleaseAndVersion(orchType, orchRel, orchVer string, hasWindows
 	return version
 }
 
-// IsKubernetesVersionGe returns if a semver string is >= to a compare-against semver string (suppport "-" suffixes)
+// IsKubernetesVersionGe returns true if actualVersion is greater than or equal to version
 func IsKubernetesVersionGe(actualVersion, version string) bool {
-	orchestratorVersion, _ := semver.NewVersion(strings.Split(actualVersion, "-")[0]) // to account for -alpha and -beta suffixes
```

Comment:
we don't need to do this anymore because the new semver package handles pre-release versions